core/templates/dev/head/pages/exploration_editor/history_tab/CompareVersionsService.js

```diff
@@ -14,123 +14,15 @@
 
 /**
  * @fileoverview Services for the exploration history tab.
+ * Service for handling all interactions with the version history tree.
```

Comment:
I think now this file is a service to compare versions of the exploration, right?